src/java/org/apache/cassandra/service/CassandraDaemon.java

```diff
@@ -366,9 +371,10 @@ protected void setup()
         // Clean up system.size_estimates entries left lying around from missed keyspace drops (CASSANDRA-14905)
         StorageService.instance.cleanupSizeEstimates();
 
-        // schedule periodic dumps of table size estimates into SystemKeyspace.SIZE_ESTIMATES_CF
-        // set cassandra.size_recorder_interval to 0 to disable
-        int sizeRecorderInterval = Integer.getInteger("cassandra.size_recorder_interval", 5 * 60);
+        logger.info("Schedule periodic dumps of table size estimates into SystemKeyspace.SIZE_ESTIMATES_CF. Set -D{}=0 to disable.",
```

Comment:
SystemKeyspace.SIZE_ESTIMATES_CF is pretty much non-telling, what table is that? A user does not have a clue. This is just a constant for him without knowing what table that actually is.

More to it, if you look into SystemKeyspace, that constant is annotated as deprecated. I would keep it like it was otherwise we are informing a user about something which is deprecate already which is quite confusing.

Reply:
Fixed.